moped-toolbox/migrate_personnel/secrets.py

```diff
@@ -0,0 +1,8 @@
+HASURA_AUTH = {
+    "hasura_graphql_endpoint": {
+        "local": "http://localhost:8080/v1/graphql",
+    },
+    "hasura_graphql_admin_secret": {
+        "local": "DuMmyApiKeyHFVOVto19otC1wX6sP2N0VSKrCD70L10B7Sm525ZR6L672i2F79M9!",
+    },
+}
```

Comment:
I wrote up [some comments for Rose](https://github.com/cityofaustin/atd-vz-data/pull/1118#discussion_r1001949543) in a PR the other day that I think apply here as well. But otherwise, I think this is fine, as it was with hers, as this is in the toolbox, but in the future, a template of the secrets file may be safer.

Reply:
thanks @frankhereford - i went ahead and ignored this file for all toolbox scripts 🔒 
https://github.com/cityofaustin/atd-moped/pull/834/commits/37ecba2c88307f947aad8b3501a0eb402ec00d97